src/main/java/dan200/computercraft/shared/peripheral/modem/TileCable.java

```diff
@@ -349,12 +349,18 @@ public void onNeighbourChange()
         {
            
             EnumFacing dir = getDirection();
-            if( !worldObj.isSideSolid(
+            if( worldObj.isAirBlock(getPos().offset( dir )) || ( !worldObj.isSideSolid(
             getPos().offset( dir ),
             dir.getOpposite()
-            ) && PeripheralUtil.getPeripheral( worldObj, getPos().offset( dir ), dir.getOpposite() ) == null
+            ) && PeripheralUtil.getPeripheral( worldObj, getPos().offset( dir ), dir.getOpposite() ) == null )
             )
             {
+                if( m_peripheralAccessAllowed == true ) //Make sure to disconnect peripheral when breaking
```

Comment:
I'm pretty sure this can be shifted into the `WiredModemWithCable` section - normal `WiredModem`s cannot have a peripheral.